### ISSUE TITLE
Function with cond is "too complex"

### ISSUE THREAD

**Issue by fchabouis** [rrrene/credo#894]
### Environment

* Credo version (`mix credo -v`): 1.5.6-ref.master.60c84ec0+uncommittedchanges
* Erlang/Elixir version (`elixir -v`): Elixir 1.12.2 (compiled with Erlang/OTP 24)
* Operating system: Ubuntu 20.04

### What were you trying to do?

This function gives me a "function is too complex" message :

```elixir
  def formated_format(resource, type, is_community_resource) do
    format = Map.get(resource, "format", "")

    cond do
      is_gtfs_rt?(format) -> "gtfs-rt"
      is_netex?(format) -> "NeTEx"
      is_neptune?(format) -> "Neptune"
      is_gtfs?(format) -> "GTFS"
      is_siri_lite?(format) -> "SIRI Lite"
      is_siri?(format) -> "SIRI"
      type == "public-transit" and not is_community_resource -> "GTFS"
      true -> format
    end
  end
```

```
Function is too complex (CC is 10, max is 9).
```
I don't see a better way to express this though. 

Is this behavior expected ?

**Comment by rrrene** [comment]
> Is this behavior expected ?

Yeah, I am pretty sure it is. 

It's just that CC is not a perfect metric, but an academic one to highlight code points with lots of branching. So this is "correct" in reporting this piece of code, but it is up to you to decide whether to (a) deactivate this check for this project or (b) configure this check to be more suitable to your needs.

TL;DR There's nothing wrong with your code or the check, the two might just not be made for each other. 😉 👍

**Comment by kelvinst** [comment]
Also worth mentioning that you could use `# credo:disable-for-next-line Credo.Check.Refactor.CyclomaticComplexity` before the function head and Credo will disable this specific check there.

**Comment by fchabouis** [comment]
Thanks for tour replies !
I juste wanted to check I wasn't missing an obviously more elegant solution to write this piece of code.
